frontend/containers/AuctionsFakeData.vue

```diff
@@ -147,6 +147,7 @@ export default Vue.extend({
         restart() {
             if (this.selectedAuction) {
                 this.selectedAuction.isActive = true;
+                // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
I read your conversation history about these lines. I have one more idea:

We write our own custom wrapper for the AntDesign messenger, that we can then call in all of the other places we currently use the message component. 

We can then disable the typescript check only in the helper, which removes the need to add these comments everywhere in the rest of our codebase. What do you think of this idea @valiafetisov?

Reply:
Yeah, sounds a little cleaner!